i18n/en.json

```diff
@@ -1508,6 +1508,10 @@
     "id": "api.file.test_connection.s3.bucked_create.app_error",
     "translation": "Unable to create bucket."
   },
+  {
+    "id": "api.file.reader.reading_local.app_error",
+    "translation": "Encountered an error opening a reader from local server storage"
```

Comment:
do you mean local file storage?

Reply:
I used the same text found other places in the file. Actually, we the most common pattern is "local server file storage"

Reply:
Cool, do we want to use `local server file storage` in that case?

Reply:
Submitted a proposal that changes this text to use the most commonly used, and replaced the other occurrence of "local server storage"